app/controllers/support/providers/accredited_partnerships_controller.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+module Support
+  module Providers
+    class AccreditedPartnershipsController < ApplicationController
+      include ClearStashable
```

Comment:
One line calling to clear the stash from the form, do we need the module even if is only one call to each controller?